packages/subcommands/README.md

```diff
@@ -79,23 +96,37 @@ module.exports = class UserCommand extends SubCommandPluginCommand {
 	constructor(context, options) {
 		super(context, {
 			...options,
-			subCommands: ['add', 'remove', 'list', 'reset', { input: 'show', default: true }]
+			subcommands: [
+				new MessageSubcommandMappings([
+					{ name: 'add', to: 'messageAdd' },
+					{ name: 'remove', to: 'messageRemove' },
+					{ name: 'list', to: 'messageList', default: true }
+				]),
+				new ChatInputSubcommandMappings([
+					{ name: 'add', to: 'chatInputAdd' },
+					{ name: 'remove', to: 'chatInputRemove' },
+					{ name: 'list', to: 'chatInputList' }
+				])
```

Comment:
```suggestion
        {
          name: 'add',
          messageRun: 'messageAdd',
          chatInputRun: 'chatInputAdd'
        },
        {
          name: 'remove',
          messageRun: 'messageRemove',
          chatInputRun: 'chatInputRemove'
        },
        {
          name: 'list',
          messageRun: 'messageList',
          chatInputRun: 'chatInputList'
        }
```

Reply:
just ignore the indentation as of now, ill fix it in some random pr

Reply:
Or just run `yarn prettier --write .`

Reply:
Nah I did this edit on raw github, I do have a pr on 24's fork I'll fix the indentation there